src/app/cic/controllers/checkDetails.js

```diff
@@ -25,10 +24,6 @@ class CheckDetailsController extends DateController {
       callback(err, locals);
     });
   }
-
-  next() {
```

Comment:
No need for functions as [next can be a relative string path](https://github.com/HMPO/hmpo-form-wizard/tree/master#next-steps)